packages/outline-view/src/browser/outline-view-widget.ts

```diff
@@ -1,9 +1,9 @@
 /*
-* Copyright (C) 2017 TypeFox and others.
-*
-* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-*/
+ * Copyright (C) 2017 Ericsson and others.
```

Comment:
Should be  `TypeFox`

Reply:
Wasn't it suposed to be Ericsson when someone other than you, kittaakos or svenefftinge first commited?

Reply:
my fault, sorry, I forgot jbicker and AlexTugarev

if the copyright is present but missing spaces then only spaces should be added, the content should remain untouched

Reply:
Alright! Will be pushing my changes